{"uid":"35761d25b26ebf97","name":"test_get_inventDash_details","fullName":"Test_05_InventDashboard.TestInventDashboard#test_get_inventDash_details","historyId":"61f06f8516f15b795953d450f19978a0","time":{"start":1692338449817,"stop":1692338464483,"duration":14666},"description":"Checking Dashboard data","descriptionHtml":"<p>Checking Dashboard data</p>\n","status":"failed","statusMessage":"AssertionError: False != True : Verification Failed to mismatch data.","statusTrace":"self = <TestsCases.Test_05_InventDashboard.TestInventDashboard testMethod=test_get_inventDash_details>\n\n    @allure.description(f\"Checking Dashboard data\")\n    @allure.severity(allure.severity_level.NORMAL)\n    def test_get_inventDash_details(self):\n        self.LoginMethod.nativelogin(\"frugal@latido.com.np\", \"Test@123\")\n        try:\n            self.assertEqual(self.InventDash.execute_info_gather(), True, msg=\"Verification Failed to mismatch data.\")\n        # Checking if assertion failed\n        except (NoSuchElementException, AssertionError, TimeoutException) as e:\n            allure.attach(self.driver.get_screenshot_as_png(), name=\"Failed_for_customer\", attachment_type=AttachmentType.PNG)\n>           raise e\n\nTest_05_InventDashboard.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nTest_05_InventDashboard.py:33: in test_get_inventDash_details\n    self.assertEqual(self.InventDash.execute_info_gather(), True, msg=\"Verification Failed to mismatch data.\")\nC:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.3.3\\plugins\\python-ce\\helpers\\pycharm\\teamcity\\diff_tools.py:33: in _patched_equals\n    old(self, first, second, msg)\nE   AssertionError: False != True : Verification Failed to mismatch data.","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_TestInventDashboard","time":{"start":1692338449817,"stop":1692338449817,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"stepsCount":0}],"testStage":{"description":"Checking Dashboard data","status":"failed","statusMessage":"AssertionError: False != True : Verification Failed to mismatch data.","statusTrace":"self = <TestsCases.Test_05_InventDashboard.TestInventDashboard testMethod=test_get_inventDash_details>\n\n    @allure.description(f\"Checking Dashboard data\")\n    @allure.severity(allure.severity_level.NORMAL)\n    def test_get_inventDash_details(self):\n        self.LoginMethod.nativelogin(\"frugal@latido.com.np\", \"Test@123\")\n        try:\n            self.assertEqual(self.InventDash.execute_info_gather(), True, msg=\"Verification Failed to mismatch data.\")\n        # Checking if assertion failed\n        except (NoSuchElementException, AssertionError, TimeoutException) as e:\n            allure.attach(self.driver.get_screenshot_as_png(), name=\"Failed_for_customer\", attachment_type=AttachmentType.PNG)\n>           raise e\n\nTest_05_InventDashboard.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nTest_05_InventDashboard.py:33: in test_get_inventDash_details\n    self.assertEqual(self.InventDash.execute_info_gather(), True, msg=\"Verification Failed to mismatch data.\")\nC:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.3.3\\plugins\\python-ce\\helpers\\pycharm\\teamcity\\diff_tools.py:33: in _patched_equals\n    old(self, first, second, msg)\nE   AssertionError: False != True : Verification Failed to mismatch data.","steps":[],"attachments":[{"uid":"51e1d582ecf9ca46","name":"Failed_for_customer","source":"51e1d582ecf9ca46.png","type":"image/png","size":182380}],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"stepsCount":0},"afterStages":[{"name":"_unittest_setUpClass_fixture_TestInventDashboard::0","time":{"start":1692338496844,"stop":1692338496844,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"stepsCount":0}],"labels":[{"name":"feature","value":"Inventory Dashboard"},{"name":"severity","value":"normal"},{"name":"suite","value":"Test_05_InventDashboard"},{"name":"subSuite","value":"TestInventDashboard"},{"name":"host","value":"0day"},{"name":"thread","value":"14848-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"Test_05_InventDashboard"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":4,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":4},"items":[{"uid":"f7098ec06c0d9a50","status":"failed","statusDetails":"AssertionError: False != True : Verification Failed to mismatch data.","time":{"start":1691733904356,"stop":1691733919777,"duration":15421}},{"uid":"6fd2b3b36a455081","status":"failed","statusDetails":"AssertionError: False != True : Verification Failed to mismatch data.","time":{"start":1691124263960,"stop":1691124278598,"duration":14638}},{"uid":"3adfde4d227534b0","status":"failed","statusDetails":"AssertionError: False != True : Verification Failed to mismatch data.","time":{"start":1690520507533,"stop":1690520518225,"duration":10692}}]},"tags":[]},"source":"35761d25b26ebf97.json","parameterValues":[]}